evidence/pool.go

```diff
@@ -165,17 +165,12 @@ func (evpool *Pool) AddEvidence(evidence types.Evidence) error {
 			return fmt.Errorf("failed to verify %v: %w", ev, err)
 		}
 
-		// 2) Compute priority.
```

Comment:
was this only used for the outqueue?

Reply:
Initially, I believe the idea was to propose evidence in order of validator power, so a malicious validator with more power was processed first. This sort of makes sense expect it doesn't take into consideration coalitions i.e. multiple validators colluding together and secondly it's implementation was bypassed so instead, at least for now, I have made the order based on age so that the oldest evidence is submitted first